src/DotNetBoilerplate.Application/Reviews/Browse/BrowseReviewsByBookIdHandler.cs

```diff
@@ -0,0 +1,19 @@
+using DotNetBoilerplate.Application.Reviews.DTO;
+using DotNetBoilerplate.Core.Reviews;
+using DotNetBoilerplate.Shared.Abstractions.Queries;
+
+namespace DotNetBoilerplate.Application.Reviews.Browse;
+
+internal sealed class BrowseReviewsByBookIdHandler(
```

Comment:
Spróbować sobie to zrobić w jednym endpoincie, i jak chcemy wyszukać review dla danej książki to zawężyć filtr o id książki z query params /reviews?bookId=1